Atomic/syscontainers.py

```diff
@@ -1114,7 +1133,9 @@ def update_container(self, name, setvalues=None, rebase=None):
         if not repo:
             raise ValueError("Cannot find a configured OSTree repo")
 
-        path = os.path.join(self._get_system_checkout_path(), name)
+        checkout = self._get_system_checkout_path()
```

Comment:
could you rename it to `system_checkout_path`?

Reply:
yup, sure, thanks for the review =)